assets/dev/js/editor/elements/views/base.js

```diff
@@ -659,6 +673,35 @@ BaseElementView = BaseContainer.extend( {
 		this.renderHTML();
 	},
 
+	isAtomicDynamic( settingKey ) {
+		const widgetConfig = elementor.widgetsCache[ this.model.config.name ]
+		return settingKey.includes( '__dynamic__' ) &&
+			elementorCommon.config.experimentalFeatures.e_nested_atomic_repeaters &&
+			widgetConfig.supports_atomic_dynamic_titles; // May be redundant
+	},
+
+	getDynamicValue( settings ) {
+		const valueToParse = settings.attributes.__dynamic__.item_title,
+			dynamicSettings = { active: true };
```

Comment:
I think that `dynamicSettings ` should be retrieved from somewhere. 
I couldn't find where it is coming from.